### DISCUSSION TITLE
Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by stephanlj** [brutella/hkknx-public#300] [Q&A]
Hi,
the current software does not function on the pi 5. As a system it requires bookworm. Is it possible to install bookworm and then manually install home kit???

**Comment by brutella** [comment]
Yes, you should be able to install and run the 32-bit Raspberry Pi version of hkknx as explained in [instructions](https://hochgatterer.me/hkknx/docs/install/). Do not use the Raspberry Pi image but the binary.
